src/Sql/ActionableInsights/dbo/Stored Procedures/Report/Report_Create.sql

```diff
@@ -0,0 +1,36 @@
+CREATE PROCEDURE [dbo].[Report_Create]
+    @Id UNIQUEIDENTIFIER OUTPUT,
+    @OrganizationId UNIQUEIDENTIFIER,
+    @Name NVARCHAR(MAX),
+    @GroupId UNIQUEIDENTIFIER,
+    @Type TINYINT,
+    @Parameters NVARCHAR(MAX),
```

Comment:
⚠️ I am not sure what this is ... is there something Architecture can read and reference here?